website/docs/markdown-features.mdx

```diff
@@ -1,3 +1,10 @@
+<!--
```

Comment:
We shouldn't be adding these comments to MDX. Refer to D21496076, I've whitelisted the MDX extension.